packages/redux/src/wishlists/selectors/wishlistsSets.ts

```diff
@@ -132,6 +132,24 @@ export const isWishlistSetLoading = (
 ): boolean | undefined =>
   fromWishlistSetsReducer.getIsSetLoading(state.wishlist.sets)[setId];
 
+/**
+ * Returns the fetched status of a specific wishlist set.
+ *
+ * @memberof module:wishlists/selectors
+ *
+ * @param {object} state - Application state.
+ * @param {string} setId - Global identifier of the set in the wishlist.
+ *
+ * @returns {boolean} If a certain set has been fetched or not.
+ */
+export const isWishlistSetFetched = (
+  state: StoreState,
+  setId: WishlistSet['setId'],
+): boolean =>
+  !!setId &&
```

Comment:
Here I'd probably do something similar to other selectors like the fittings example

```js
getIsLoading(state.products.fittings).hasOwnProperty(id) && 
areProductFittingsLoading(state, id) === false;
```

Because in fact, your last verification (`!!getWishlistSet(state, setId)`) alone does that, if you can get the wishlist set, it was fetched. (Isn't it too much to check if something is fetched? Can't you have an empty object being returned for some reason, thus turning this a false positive? 🤔 )

I don't know, it's more a consistency thing... And for example, why do you need to verify `!!setId` if the argument is not optional? If its value is undefined for some reason, the `isWishlistSetLoading` already returns the correct value for that, you know what I mean?